developers/wcs/tools/collections-tool.mdx

```diff
@@ -1,77 +1,116 @@
 
+### Edit the General section
 
-In this example, the WCD Dashboard is connected to an external instance as well as the two WCD instances.
+Use the `General` section for collection wide settings.
 
-<center><img src={WCDConnectedInstances} width="100%" alt="Connected instances"/></center>
+<center><img src={WCDEdGen} width="75%" alt="General edit section"/></center>
 
-The GraphQL console can query any of the instances.
+#### Name the collection.
 
-To select a cluster, click the `Query` button and chose a cluster from the list that appears.
+The collection name must begin with a capital letter.
 
-### Authentication
+The name can only contain letters, numbers, and the under-score (_) character. Spaces are not allowed.
 
-The query app connects to WCD instances, sandboxes and serverless clusters, without any additional authentication details. The query app can also connect to external Weaviate instances if you have anonymous access enabled or authentication credentials.
+#### Select multi-tenancy
 
-### Pass authentication details
+[Multi-tenant collections](/developers/weaviate/manage-data/multi-tenancy) are useful if you have multiple users or need data isolation. By default, collections are single tenant. Set the `Multi-tenancy` toggle to enable multi-tenancy.
 
-import ExternalAuth from '/_includes/wcs/query-auth-details.mdx';
+#### Set a default vectorizer
 
-<ExternalAuth/>
+Set a [vectorizer](/developers/weaviate/config-refs/schema#vectorizer) for the collection. Weaviate uses the default vectorizer when you import unvectorized data and when Weaviate queries external APIs that require vectorized inputs.
 
-### Pass inference keys
+The `Vectorizer` setting is a drop-down list. After you select a vectorizer, the editor expands to let you configure the model.
 
-To pass API keys for inference modules, use request headers. The `Header` tab is at bottom of the query screen. Add the service inference key to the query headers.
+Switch the `Vectorize class name` toggle if you don't want to create a vector for the class name.
 
-In this example, the `X-OpenAI-Api-Key` header sets the OpenAI API key.
+### Edit the Properties section
 
-<center><img src={WCDInferenceApiKey} width="100%" alt="How to add an API inference key to a WCD console query"/></center>
+Use the `Properties` section to fine-tune individual properties in your collection.
 
-The header can pass multiple keys.
+#### Add properties to the collection
 
-<center><img src={WCDManyInferenceApiKeys} width="80%" alt="How to add multiple API inference keys to a WCD console query"/></center>
+Enter property names into the properties editor.
+
+The property names must begin with a lowercase letter.
+
+The names can only contain letters, numbers, and the under-score (_) character. Spaces are not allowed.
+
+<center><img src={WCDEdProp} width="75%" alt="Properties edit section"/></center>
+
+#### Configure additional details
```

Comment:
It might be helpful to describe how to add nested properties when the property type is `Object`.